src/qt/bitcoin.cpp

```diff
@@ -353,15 +409,22 @@ int main(int argc, char *argv[])
         return EXIT_FAILURE;
     }
 
-    if (!ReadConfigFile(mapArgs, mapMultiArgs)) {
+    // Reread config file after correct chain is selected
+    if (!gArgs.ReadConfigFiles(error_msg, true)) {
         ThreadSafeMessageBox(strprintf("Error reading configuration file.\n"),
                 "", CClientUIInterface::ICON_ERROR | CClientUIInterface::OK | CClientUIInterface::MODAL);
         QMessageBox::critical(nullptr, PACKAGE_NAME,
             QObject::tr("Error: Cannot parse configuration file."));
```

Comment:
```suggestion
    if (!gArgs.ReadConfigFiles(error, true)) {
        ThreadSafeMessageBox(strprintf("Error reading configuration file: %s\n", error),
                "", CClientUIInterface::ICON_ERROR | CClientUIInterface::OK | CClientUIInterface::MODAL);
        QMessageBox::critical(nullptr, PACKAGE_NAME,
            QObject::tr("Error: Cannot parse configuration file: %1.").arg(QString::fromStdString(error)));
```

I think we can show the error message, no?

Reply:
Ok.